cms/static/js/collections/asset.js

```diff
@@ -33,6 +33,44 @@ define(["backbone.paginator", "js/models/asset"], function(BackbonePaginator, As
             this.currentPage = currentPage;
             this.start = start;
             return response.assets;
+        },
+
+        setPage: function (page) {
+            var oldPage = this.currentPage,
+                self = this;
+            this.goTo(page - 1, {
+                reset: true,
+                success: function () {
+                    self.trigger('page_changed');
+                },
+                error: function () {
+                    self.currentPage = oldPage;
+                }
+            });
+        },
+
+        nextPage: function () {
+            if (this.currentPage < this.totalPages - 1) {
+                this.setPage(this.currentOneIndexPage() + 1);
+            }
+        },
+
+        previousPage: function () {
+            if (this.currentPage > 0) {
+                this.setPage(this.currentOneIndexPage() - 1);
+            }
+        },
+
+        currentOneIndexPage: function () {
+            return this.currentPage + 1;
```

Comment:
Correct. `asset.js` is used exclusively with the assets API endpoint, so in this case it's a safe assumption.